flybywiresim/a32nx
# [CDU] Bugfixes to INIT and FUEL PRED pages

Fixes #2515
Fixes #2507 

## Summary of Changes

- [x] Merge alternate and trip coefficient tables 
- [x] Fix error message logic
- [x] add EFOB to FPLAN
- [x] Add logic to check entered reserved weight isn't beyond percent limt
- [x] Try to re-work pred-failure or remove it


Discord username (if different from GitHub): Lucky38i

## Testing instructions

- Test trips between 20NM - 3100NM and FL100 - FL 390, verify you get predictions for anything in those parameters
- Check EFOB shows in FPLAN page, AFTER doing your INIT B, verify it's dashed before
- Try to enter a reserve weight that's 15% above the trip weight, you should get dashes results signify out of range, no error prints.
- Try to test any other bugs you're aware about regarding INIT and FUEL PRED.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
